libtransmission/announcer-udp.cc

```diff
@@ -675,9 +771,8 @@ class tr_announcer_udp_impl final : public tr_announcer_udp
                     it != std::end(reqs))
                 {
                     logtrace(tracker.log_name(), fmt::format("{} is an announce request!", transaction_id));
-                    auto req = *it;
-                    it = reqs.erase(it);
-                    req.onResponse(action_id, buf);
+                    it->on_response(ip_protocol, action_id, buf);
```

Comment:
This line is within an if condition that checks for this.

Reply:
> I could be blind, but which one of the two `if`: line 765 or line 767? And how is the check done?

My bad my bad. I somehow thought you were talking about line 760...

> which isn't a big deal, but it will give a response with neither `pex` nor `pex6`

Yes, that's exactly the behaviour I'm going for. We might not be able to parse the peers, but we at least can scavenge some tracker stats from the response.